.changelog/1827.bugfix.md

```diff
@@ -0,0 +1,8 @@
+i18n: Update translations from Transifex and add normalization script
```

Comment:
This should automatically combine with https://github.com/oasisprotocol/oasis-wallet-web/pull/1815